docs/guides/how-to-contribute-algorithms.md

```diff
@@ -1 +1,21 @@
-# How to contributrbute an algorithm to Ianvs
\ No newline at end of file
+# How to contributrbute an algorithm to Ianvs
+
+
+
+Before start, it should be known that Ianvs testing algorithm development depends on [Sedna Lib]. The following is recommended contribute workflow:
```

Comment:
Comment is needed to state that:

Ianvs serves as testing tools for test objects, e.g., algorithms. Ianvs does NOT include code directly on test obejct.
Alogorithms serve as typical test objects in Ianvs and detailed algorithms are thus NOT included in this Ianvs python file.
As for the details of example test obejcts, e.g., algorithms, please refer to third party packages in Ianvs example. For example, AI workflow and interface pls refer to sedna and module implementation pls refer to third party package like FPN_TensorFlow and Sedna IBT algortihm.

Reply:
1. release a repo independent of sedna and ianvs, but interface should still follow the sig ai algorithm interface to launch ianvs. 
2. intergrated into sedna so that ianvs can use directly. in this case, you can connect sedna owners for help.